sample-apps/react/react-video-demo/package.json

```diff
@@ -5,6 +5,7 @@
   "type": "module",
   "scripts": {
     "dev": "vite",
+    "start": "vite",
```

Comment:
I take it start is the common one used in stream?

Reply:
Yes, typically all of our packages have a default `start` script.